modules/n1ql/pages/n1ql-language-reference/selectintro.adoc

```diff
@@ -448,76 +441,76 @@ In this example:
 SELECT DISTINCT route.sourceairport,
                 route.airlineid,
                 airline[0].callsign
-FROM `travel-sample` route NEST `travel-sample` airline
+FROM `travel-sample`.inventory.route
+NEST `travel-sample`.inventory.airline
   ON KEYS route.airlineid
-WHERE route.type = "route"
-  AND route.airline = "AA"
+WHERE route.airline = "AA"
 LIMIT 4;
 ----
 
 .Results
 [source,JSON]
 ----
 [
-   {
-      "airlineid": "airline_24",
-      "callsign": "AMERICAN",
-      "sourceairport": "ITH"
-   },
-   {
-      "airlineid": "airline_24",
-      "callsign": "AMERICAN",
-      "sourceairport": "WAW"
-   },
-   {
-      "airlineid": "airline_24",
-      "callsign": "AMERICAN",
-      "sourceairport": "BKK"
-   },
-   {
-      "airlineid": "airline_24",
-      "callsign": "AMERICAN",
-      "sourceairport": "GGT"
-   }
+  {
+    "airlineid": "airline_24",
+    "callsign": "AMERICAN",
+    "sourceairport": "ABE"
```

Comment:
why have the source airports changed? new data?
Ah, because there's no sort order specified, and the implicit one has changed because of new PLAN?

Reply:
When I tested the original query (with the default scope and collection) I got the new results. In a few cases, unfortunately, the results in the docs were just wrong. I'm testing them all as I updated them.

Reply:
Great, yes, it's really nice to see how this fits in with the docs/QE work!